cumulusci/tasks/bulkdata/snowfakery.py

```diff
@@ -328,8 +362,6 @@ def tick(
         ):  # pragma: no cover
             self.logger.info("Finished Generating")
         # queue is full
-        elif self.data_gen_q.full:
-            self.logger.info("Waiting before datagen (queue full)")
```

Comment:
Why is this not needed? (and in any case you need to remove the comment on line 364)